{{cookiecutter.project_name}}/.github/workflows/build.yaml

```diff
@@ -15,15 +15,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python 3.12
-        uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
-          cache: "pip"
-          cache-dependency-path: "**/pyproject.toml"
-      - name: Install build dependencies
-        run: python -m pip install --upgrade pip wheel twine build
+          filter: blob:none
+          fetch-depth: 0
```

Comment:
Should we include filter: `blob:none` by default in all action scripts or rather stick with the default shallow checkout?

Reply:
As far as I can tell, `filter: blob:none` is simply a better default. There are scenarios where one would want to do a full clone, but I can’t think of a scenario where `fetch-depth: 1` would be in any way preferable.

Reply:
I think `blob:none` is a good idea in our workflows.